Fix #13304: Add documentation about fonts and correct error message (#13305)
OpenTTD/OpenTTD: docs/fonts.md

# Fonts for OpenTTD

> OpenTTD needs four different fonts:

Reviewer: ```suggestion
OpenTTD uses four different fonts:
```

- a large font (used for news headlines etc)
- a monospace font (used for text files such as NewGRF readmes).

You can use the following types of fonts with OpenTTD:

## OpenTTD's default fonts
These fonts are OpenTTD Sans (small and medium), OpenTTD Serif (large) and OpenTTD Mono (monospace). They are distributed as part of OpenTTD since version 14. The font files are included in the baseset directory of OpenTTD.

These fonts are active by default and support the Latin, Greek and Cyrillic scripts at present.

## Traditional sprite fonts
These are the classic bitmap fonts included as part of the base graphics. They support only the Latin script.

These fonts can be activated in the Graphics section of the Game options window, by enabling the option "Use traditional sprite fonts".

## System fonts
These are fonts installed on your computer. OpenTTD tries to automatically detect and activate a suitable system font in case you have selected a language not supported by the default fonts. However, if this fails, you may have to set a font manually.

There are two ways to manually set system fonts, using the `font` console command or editing the openttd.cfg file.

### Using the console

> Open the console. On a normal English keyboard this is done by pressing the ~ key (to the left of 1).

Reviewer: This almost sounds like non-English keyboards are "not normal" 😉 

```suggestion
Open the console. On most English keyboards this is done by pressing the ~ key (to the left of 1).
```

Author: Rephrased slightly because:
- we force this to be the key to the left of 1 (scancode 29) on all keyboard layouts, not just English: https://github.com/OpenTTD/OpenTTD/blob/23dcb4f4a70849055e5f7ff87e89df98344d2581/src/video/win32_v.cpp#L629 or https://github.com/OpenTTD/OpenTTD/blob/23dcb4f4a70849055e5f7ff87e89df98344d2581/src/video/win32_v.cpp#L629
- UK English keyboards don't have ~ on that key, but rather \` and ¬, so "most" would be wrong.

> The command to change a font is `font [medium|small|large|mono] [<font name>] [<size>]`.

Reviewer: It would be nice to have an example command here, to help people parse the syntax.

# Fonts for OpenTTD
OpenTTD uses four different fonts:
- a medium font (used for most texts in the game)
- a small font (used for the smallmap legend etc)
- a large font (used for news headlines etc)
- a monospace font (used for text files such as NewGRF readmes).

You can use the following types of fonts with OpenTTD:

> ## OpenTTD's default fonts

Reviewer 2: As with the other markdown files, please add a newline before and after a new title. Consistency goes a long way ;)
